vscode-trace-common/src/client/tsp-client-provider-impl.ts

```diff
@@ -1,9 +1,10 @@
-import { TspClient } from 'tsp-typescript-client/lib/protocol/tsp-client';
-import { RestClient } from 'tsp-typescript-client/lib/protocol/rest-client';
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Maybe a leftover? I do not see "any" used in this file?

Reply:
yep, it was needed before. I remove it.